### DISCUSSION TITLE
Support indented code chunks in lists

### DISCUSSION THREAD

**Discussion by homerhanumat** [quarto-dev/quarto-cli#11502] [Feature Requests]
### Description

In R Markdown, indented code chunks in a list knit up nicely, as in:

````
---
title: "test"
output: html_document
---

1. Here is some R-code:

    ```{r}
    1 + 1
    ```

1. Here is some more R-code:

    ```{r}
    plot(cars)
    ```
````

However, the similar .qmd file:

````
---
title: "test"
format: html
---

1. Here is some R-code:

    ```{r}
    1 + 1
    ```

1. Hwre is some more R-code:

    ```{r}
    plot(cars)
    ```
````

shows `::: {.cell}`, and `::: {.cell-output .cell-output-stdout}` when rendered.

**Comment by mcanouil** [comment]
The main issue this that your code cells are not properly indented.

- https://github.com/quarto-dev/quarto-cli/issues/11021


Also why, it's not really supported:

- https://github.com/quarto-dev/quarto-cli/issues/9970


Finally, the issue previously reported:

- https://github.com/quarto-dev/quarto-cli/issues/7551

I am going to close this as a duplicate.
